src/EnergyPlus/ScheduleManager.hh

```diff
@@ -48,6 +48,9 @@
 #ifndef ScheduleManager_hh_INCLUDED
 #define ScheduleManager_hh_INCLUDED
 
+// C++ Headers
+#include <map>
+
```

Comment:
Was there a reason you needed to include this here?  I don't see any of your changes needing it.  If you need it in the .cc file, go ahead and put it over there, not in the .hh file.

Reply:
Right.